pex/testing.py

```diff
@@ -213,13 +213,9 @@ def mock_logger(msg, v=None):
 
 # TODO(wickman) Why not PEX.run?
 def run_simple_pex(pex, args=(), env=None):
-  po = subprocess.Popen(
-      [sys.executable, pex] + list(args),
-      stdout=subprocess.PIPE,
-      stderr=subprocess.STDOUT,
-      env=env)
-  po.wait()
-  return po.stdout.read().replace(b'\r', b''), po.returncode
+  process = Executor.open_process([sys.executable, pex] + list(args), env=env, combined=True)
+  stdout, stderr = process.communicate()
```

Comment:
With combined=True, stderr will == stdout, right? might be clearer just to drop it here.